File: app/src/main/java/piotr/michalkiewicz/mealplannerclient/view/personalization/StartCustomizationActivity.java
package piotr.michalkiewicz.mealplannerclient.view.personalization;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import com.mealplannerclient.R;

import piotr.michalkiewicz.mealplannerclient.view.main_menu.MainMenuActivity;

public class StartCustomizationActivity extends AppCompatActivity {

    private Button startCustomizationBtn;
    private Button skipCustomizationBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_start_customization);

        startCustomizationBtn = findViewById(R.id.startCustomizationBtn);
        skipCustomizationBtn = findViewById(R.id.skipCustomizationBtn);

        startCustomizationBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // TODO Start customization here
            }
        });

        skipCustomizationBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent myIntent = new Intent(StartCustomizationActivity.this, MainMenuActivity.class);
                startActivity(myIntent);
            }
        });

    }
}

File: app/src/main/java/piotr/michalkiewicz/mealplannerclient/view/personalization/StartCustomizationActivity.kt
package piotr.michalkiewicz.mealplannerclient.view.personalization

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import piotr.michalkiewicz.mealplannerclient.R
import piotr.michalkiewicz.mealplannerclient.databinding.ActivityLoginBinding
import piotr.michalkiewicz.mealplannerclient.databinding.ActivityStartCustomizationBinding
import piotr.michalkiewicz.mealplannerclient.view.menu.MainMenuActivity

class StartCustomizationActivity : AppCompatActivity() {
    private lateinit var binding: ActivityStartCustomizationBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityStartCustomizationBinding.inflate(layoutInflater)

        binding.startCustomizationContent.startCustomizationBtn.setOnClickListener(View.OnClickListener {
            // TODO Start customization here
        })
        binding.startCustomizationContent.skipCustomizationBtn.setOnClickListener(View.OnClickListener {
            val myIntent = Intent(this@StartCustomizationActivity, MainMenuActivity::class.java)
            startActivity(myIntent)
        })

        setContentView(binding.root)
    }
}